src/pages/Pages/index.tsx

```diff
@@ -1,63 +1,67 @@
 import { PlusOutlined } from '@ant-design/icons';
-import { Button, Tooltip, Popconfirm } from 'antd';
-import React, { useState, useRef } from 'react';
-import { useIntl, FormattedMessage } from 'umi';
+import { Button, Tooltip, Popconfirm, message } from 'antd';
+import React, { useRef } from 'react';
+import { useIntl, FormattedMessage, Link } from 'umi';
 import { PageContainer } from '@ant-design/pro-layout';
 import type { ProColumns, ActionType } from '@ant-design/pro-table';
 import ProTable from '@ant-design/pro-table';
 
-import { pages } from '@/services/escola-lms/pages';
+import { pages, deletePage } from '@/services/escola-lms/pages';
 import { DeleteOutlined, EditOutlined } from '@ant-design/icons';
-
-/*
-const handleUpdate = async (fields: API.PageListItem, id?: number) => {
-  console.log('handleUpdate', fields, id);
-  return true;
-};
-*/
+import TypeButtonDrawer from '@/components/TypeButtonDrawer';
 
 const handleRemove = async (id: number) => {
-  console.log('handleRemove', id);
-  return true;
+  return deletePage(id).then((response) => {
+    if (response.success) {
+      message.success(response.message);
+    }
+    return true;
+  });
 };
 
 const TableList: React.FC = () => {
-  const [modalVisible, setModalVisible] = useState<number | false>(false);
-  const [data, setData] = useState<API.PageListItem[]>([]);
   const actionRef = useRef<ActionType>();
   const intl = useIntl();
 
-  console.log('outupt', data, modalVisible);
-
   const columns: ProColumns<API.PageListItem>[] = [
     {
       title: <FormattedMessage id="ID" defaultMessage="ID" />,
       dataIndex: 'id',
       hideInSearch: true,
     },
     {
-      title: <FormattedMessage id="name" defaultMessage="name" />,
-      dataIndex: 'total',
-      hideInSearch: false,
+      title: <FormattedMessage id="title" defaultMessage="title" />,
+      dataIndex: 'title',
+      hideInSearch: true,
+    },
+    {
+      title: <FormattedMessage id="slug" defaultMessage="slug" />,
+      dataIndex: 'slug',
+      hideInSearch: true,
     },
     {
-      title: <FormattedMessage id="user" defaultMessage="user" />,
-      dataIndex: 'user_id',
-      hideInSearch: false,
+      title: <FormattedMessage id="author" defaultMessage="author" />,
+      dataIndex: 'author_id',
+      hideInSearch: true,
+      render: (_, record) => (
+        <TypeButtonDrawer
+          key={'user'}
+          type="EscolaLms\Core\Models\User"
+          type_id={record.author_id}
+        />
+      ),
     },
     {
       hideInSearch: true,
       title: <FormattedMessage id="pages.searchTable.titleOption" defaultMessage="操作" />,
```

Comment:
let's get rid of chinese default messages where you code something or default messages at all where not necessary

Reply:
```suggestion
      title: <FormattedMessage id="pages.searchTable.titleOption" defaultMessage="option" />,
```